carrot-log4j-aws-sns/pom.xml

```diff
@@ -51,8 +51,8 @@
 
 		<dependency>
 			<groupId>com.amazonaws</groupId>
-			<artifactId>aws-java-sdk</artifactId>
-			<version>1.4.1</version>
+			<artifactId>aws-java-sdk-sns</artifactId>
```

Comment:
Figured it made sense to drop down the level of external dependency to just the sns stuff